examples/bare-metal/explorer_board/src/platform/mic_support.h

```diff
@@ -7,12 +7,7 @@
 #include <xcore/clock.h>
 #include <xcore/port.h>
 
-#include "app_conf.h"
-
-#define MAX(X, Y) ((X) >= (Y) ? (X) : (Y))
-#define MIN(X, Y) ((X) <= (Y) ? (X) : (Y))
-
-#define MIC_DUAL_NUM_CHANNELS 2
+#include "../app_conf.h"
```

Comment:
Could you return this to a non relative path, as it will prevent portability by forcing a specific file structure.